yt_dlp/extractor/_extractors.py

```diff
@@ -135,7 +135,7 @@
 from .anvato import AnvatoIE
 from .aol import AolIE
 from .apa import APAIE
-from .aparat import AparatIE
+from .aparat import AparatIE, AparatPlaylistIE
```

Comment:
```py
from .aparat import (
    AparatIE,
    AparatPlaylistIE,
)
```
use this instead like other extractors do

Reply:
Thank you! Fixed in 322acaef8a603a54b20b6db426c90b9ff8d165f5.